CPP/sorting/bubble_sort.c

```diff
@@ -0,0 +1,35 @@
+// C program for implementation of Bubble sort
```

Comment:
Can you elaborate on the problem statement, as to what it is, and why it is better than other sorting algorithms.

Reply:
The only significant advantage that bubble sort has over most other implementations, even quicksort, but not insertion sort, is that the ability to detect that the list is sorted is efficiently built into the algorithm. Performance of bubble sort over an already-sorted list (best-case) is O(n).